src/AppInstallerCLIE2ETests/ErrorCommand.cs

```diff
@@ -14,6 +14,15 @@ namespace AppInstallerCLIE2ETests
     /// </summary>
     public class ErrorCommand
     {
+        /// <summary>
+        /// Reset settings file to avoid affecting output from error command.

```

Comment:
Why did you add this? I'm guessing changing the settings in the Configure tests was affecting this? If it's that, maybe we should also add it as a OneTimeTearDown to these tests, so that it does not affect other tests if the order ever changes. But also keep it here if these tests expect clean settings

Reply:
Yup that's right. One of the tests in the error command checks if there are 2 lines returned from the error command but if the settings file is invalid, it will return >2. I think this should be sufficient since the error command only outputs to console. It doesn't change the settings file during the tests so it won't impact other tests.